website/docs/advanced/builder.md

```diff
@@ -0,0 +1,190 @@
+
+Registration against the builder will only occur for active validators. Registration for eligible validators occurs at the beginning of the validator clients execution and at the middle of each epoch. It is important to note that the success of the API is not guaranteed, and the client will attempt registration again at the middle of each epoch.
+
+The beacon node must also be configured to enable the builder via the `--http-mev-relay` flag.
+
+:::
+
+## 2. Beacon Node: connect to the builder
+
+To use a builder the beacon node needs to start with the following configuration:
+
+- `--http-mev-relay` flag pointed to a MEV-boost instance or running relay (5. Builder: connected via relay URL).
+
+Each relay's URL will correspond to a specific network and will need to be chosen accordingly, i.e. running a beacon node on mainnet will require the mainnet relay.
+
+### Circuit breaker
+
+The circuit breaker is a safety feature for falling back to local execution when using a builder.
+This occurs when the builder or client using the builder endpoints encounters issues that cause missed blocks. 
+By default, the circuit breaker will be triggered after 3 slots are consecutively missed or 5 slots are missed in an epoch, but this can be configured through the `--max-builder-consecutive-missed-slots` and `max-builder-epoch-missed-slots` flags.
+
+## Registration cache
+
+validator registrations for Builder APIs are stored in a cache by default as of `4.0.7` instead of bolt db when starting the beacon node. The cache will enable:
+  - in-memory storage of the validator registrations, this clears all validator registrations on restart.
+This feature solves the unintended issue of wanting some validators unregistered while maintaining mev boost on others. In the future the db used to store registrations will be removed completely and the flag will no longer be required for this feature. Validator settings will be persisted on the validator client side.
+
+`--disable-registration-cache` flag can be used on the beacon node to fall back onto the using the bolt db. **note** values stored in the bolt db will not be cleared and you will not be able to unregister validators unless using the cache and restarting.
+
+### Parallel execution
+
+By default the beacon node will construct the consensus and execution portions of the beacon block in parallel to improve speed and efficiency. `--disable-build-block-parallel` flag can be added to prevent node from building in parallel and will build sequentially. 
+
+### Prioritizing local blocks
+
+`--local-block-value-boost` flag is a float64 value that provides an additional percentage to multiply the local block value. Use builder block if: `builder_bid_value * 100 > local_block_value * (local-block-value-boost + 100)`. This will encourage your setup to use the local execution if the value earned is not above your threshold, helping to mitigate censorship concerns.
```

Comment:
Maybe let's also move this to the FAQ or tell users they can check the following MEV-related flags (list them) in our parameters page